themes/default/content/docs/get-started/_index.md

```diff
@@ -43,7 +43,16 @@ Select one of the following options to learn how to deploy cloud resources using
                 </a>
             </div>
         </div>
-        <p>See <a href="{{< relref "/docs/intro/cloud-providers" >}}">Cloud Providers</a> for other clouds.</p>
+        <p class="text-sm leading-relaxed text-gray-600">See <a href="{{< relref "/docs/intro/cloud-providers" >}}">Cloud Providers</a> on how to use other supported clouds.</p>
+    </div>
+</div>
+<div class="md:flex flex-row mt-6 mb-6">
+    <div class="w-full border-solid border-t-2 border-gray-200">
+        <h3 class="no-anchor pt-4"><i class="fas fa-video pr-2"></i>Video Walkthrough</h3>
+        <p>Watch how to deploy a simple application using Pulumi.</p>
+        <div class="md:w-3/4">
+            {{< youtube id="QfJTJs24-JM" title="Introduction to Pulumi: Modern Infrastructure as Code" >}}
+        </div>
```

Comment:
Can we use the same overlay image as on the (new) homepage - and perhaps borrow the styling from there as well - the video looks nicer in that format.

Reply:
Done. Btw, the image should resolve correctly after everything else is merged.